shared/js/abp-preprocessed.es6.js

```diff
@@ -5,6 +5,7 @@
  * This will be browserifyed and turned into abp.js by running 'grunt'
  */
 abp = require('abp-filter-parser')
+const crypto = require('crypto')
```

Comment:
There is a native `window.crypto.subtle` api that obviates the need for this. Also keep in mind in the future that if you add an npm package you need to specify it in package.json. In this case tho, `window.crypto.subtle` api should do it without the need.

Reply:
@laurengarcia that was my first thought, but crypto.subtle doesn't support md5: https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/digest. Now, why we're using md5 server-side in the first place is a great question, but out of the scope of this PR.

Also, I was under the impression that the crypto package I used is included in node, so there's no need to specify anything new in package.json: https://nodejs.org/api/crypto.html#crypto_crypto ?